reconcile/openshift_rolebindings.py

```diff
@@ -112,7 +113,9 @@ def fetch_desired_state(ri, oc_map):
         for permission in permissions:
             cluster_info = permission["cluster"]
             cluster = cluster_info["name"]
-            namespace = permission["namespace"]
+            namespace_info = permission["namespace"]
+            namespace = namespace_info["name"]
```

Comment:
Wow, the usage of the word namespace in this method is quite confusing :sweat: 
What about renaming this to `perm_namespace_name`